nilakkhana/parser/extensions/deregister_standard_functions.py

```diff
@@ -0,0 +1,72 @@
+from markdown import Markdown
+from markdown.extensions import Extension
+
+
+class DeregisterStandardFunctionsExtension(Extension):
+
+    PREPROCESSORS = [
+        # 'normalize_whitespace',
+        # 'html_block',
```

Comment:
why commented out?

Reply:
are the names of the built-in processors of this library, since in our problem only `BLOCK_PROCESSORS` and `INLINE_PATTERNS` were in the way, I decided to leave the others. Why didn't I remove them at all? To have an idea of what the library is still using and, if necessary, the unnecessary properties can be uncommented, and then they will be disabled during runtime